packages/lodestar-cli/src/commands/beacon.ts

```diff
@@ -45,27 +60,51 @@ export class BeaconNodeCommand implements ICliCommand {
     generateCommanderOptions(command, BeaconNodeOptions);
   }
 
-  public async action(options: IBeaconCommandOptions, logger: ILogger): Promise<void> {
-    let conf: Partial<IBeaconNodeOptions> = {};
+  public async action(cmdOptions: IBeaconCommandOptions, logger: ILogger): Promise<void> {
+    let nodeOptions: Partial<IBeaconNodeOptions> = {};
+    //find better place for this once this cli is refactored
+    await initBLS();
 
-    if (options.loggingLevel) {
+    if (cmdOptions.loggingLevel) {
       // eslint-disable-next-line no-undef
       // @ts-ignore
-      logger.setLogLevel(LogLevel[options.loggingLevel]);
+      logger.setLogLevel(LogLevel[cmdOptions.loggingLevel]);
     }
 
     //merge config file
-    if (options.configFile) {
-      const parsedConfig = getTomlConfig(options.configFile, BeaconNodeOptions);
+    if (cmdOptions.configFile) {
+      const parsedConfig = getTomlConfig(cmdOptions.configFile, BeaconNodeOptions);
       //cli will override toml config options
-      conf = deepmerge(conf, parsedConfig) as Partial<IBeaconNodeOptions>;
+      nodeOptions = deepmerge(nodeOptions, parsedConfig) as Partial<IBeaconNodeOptions>;
     }
     //override current config with cli config
-    conf = deepmerge(conf, optionsToConfig(options, BeaconNodeOptions));
+    nodeOptions = deepmerge(nodeOptions, optionsToConfig(cmdOptions, BeaconNodeOptions));
+    const peerId = await getPeerId(cmdOptions.peerId);
+    const defaultDiscv5Opt = {
+      enr: ENR.createFromPeerId(peerId),
+      bindAddr: "/ip4/0.0.0.0/udp/5501",
+      bootEnrs: [] as ENR[]};
+    const discv5 = nodeOptions.network? Object.assign(defaultDiscv5Opt, nodeOptions.network.discv5) : defaultDiscv5Opt;
+    const libp2pOpt = nodeOptions.network? Object.assign(nodeOptions.network, {discv5}) : {discv5};
+    const libp2p = await createNodeJsLibp2p(peerId, libp2pOpt);
 
-    const config = options.preset === "minimal" ? minimalConfig : mainnetConfig;
+    const config = cmdOptions.preset === "minimal" ? minimalConfig : mainnetConfig;
+    // deposit takes time to be come active validator
+    config.params.MIN_GENESIS_ACTIVE_VALIDATOR_COUNT = 0;
 
-    this.node = new BeaconNode(conf, {config, logger});
+    // nodejs will create EthersEth1Notifier by default
+    const eth1 = cmdOptions.eth1 === "interop" ? new InteropEth1Notifier() : undefined;
+    this.node = new BeaconNode(nodeOptions, {config, logger, eth1, libp2p});
+    const depositDataRootList = config.types.DepositDataRootList.tree.defaultValue();
+    const genesisStateFilePath = cmdOptions.genesisState;
+    if (genesisStateFilePath) {
+      // genesis state from file
+      const genesisState = quickStartOptionToState(config, depositDataRootList, genesisStateFilePath);
+      await this.node.initFromKnownState(genesisState, depositDataRootList);
```

Comment:
I don't think we should support quick start params in beacon command (we now need genesisValidatorsRoot, genesis fork etc), it might be better if we just support starting from eth1 + genesis fork version and from ssz state.